releases/tools/release-2.0.0+.md

```diff
@@ -0,0 +1,23 @@
+---
+id: tools-2-0-0
+title: Multiplayer Tools 2.0.0-pre+
+description: Release notes for Multiplayer Tools including new features, updates, bug fixes, known issues, and information to help you upgrade.
+---
+
+## [2.0.0-pre.2] - 2023-05-02
+
+### General
+
+Dropped support for Unity 2020.3; the next supported version is Unity 2021.3
+Fixed `Failed to load type initialization for assembly Unity.Multiplayer.Tools.MetricTypes` runtime exception when building using Managed Stripping level set to high.
```

Comment:
Seem to be missing some bullet points here to separate these two different items